packages/lit-dev-tools-cjs/src/search/plugin.ts

```diff
@@ -55,26 +60,41 @@ export async function createSearchIndex(outputDir: '_dev' | '_site') {
     `../../../lit-dev-content/${outputDir}/searchIndex.json`
   );
   const docs: UserFacingPageData[] = await indexDocs(outputDir);
-  let idOffset = Number(docs[docs.length - 1].objectID);
+  let idOffset = Number(docs[docs.length - 1]?.objectID);
   const articles: UserFacingPageData[] = await indexArticles(
     outputDir,
     idOffset
   );
 
-  idOffset = Number(articles[articles.length - 1].objectID);
+  let lastId = Number(articles[articles.length - 1]?.objectID);
+  idOffset = isNaN(lastId) ? idOffset : lastId;
   const api: UserFacingPageData[] = await indexApi(outputDir, idOffset);
 
-  idOffset = Number(api[api.length - 1].objectID);
+  lastId = Number(api[api.length - 1]?.objectID);
+  idOffset = isNaN(lastId) ? idOffset : lastId;
```

Comment:
Would `isNaN` happen in an error case? Should `isNaN` throw instead of gracefully resetting?

Reply:
this was for when we skipped `_dev` folder. No longer needed